lib/spack/spack/installer.py

```diff
@@ -1476,7 +1497,11 @@ def install(self):
         failed_explicits = []
         exists_errors = []
 
+        self.term_title = TermTitle(len(self.build_pq))
```

Comment:
Does this need to be member data rather than being local to the function?

Reply:
You are right, a local variable is enough. Fixed.